packages/client/src/pages/ProfilePage/Profile.tsx

```diff
@@ -40,13 +33,15 @@ const Profile: FC = () => {
     navigate(`${AppRoute.PROFILE}/password`);
   };
 
-  const updateData = async (data: DataType): Promise<any> => {
-    console.log(data);
-    setEditMode(false);
+  const updateData = (data: DataType) => {
+    dispatch(fetchProfileChange(data)).then(action => {
+      if ('error' in action && action.error) return;
```

Comment:
В чём тут существенная польза от `in` проверка, и почему не достаточно просто 
```suggestion
      if (action.error) return;
```

Reply:
![image](https://user-images.githubusercontent.com/54811697/230658564-9ed40cd3-a3b9-4439-80b5-d88c4286be61.png)
ругается что в PayloadAction нет error как тут типизировать я не сообразил поэтому написал так

Reply:
раньше в коде так и было но чтобы убрать ошибку поменял на этот вариант